control/available_plugin.go

```diff
@@ -107,6 +107,11 @@ func newAvailablePlugin(resp plugin.Response, emitter gomit.Emitter, ep executab
 			}
 			ap.client = c
 		case plugin.NativeRPC:
+			log.WithFields(log.Fields{
+				"_module": "control-aplugin",
+				"_block":  "newAvailablePlugin",
+				"aplugin": ap.name,
+			}).Warning("This plugin is using a deprecated RPC protocol. Please update " + ap.name)
```

Comment:
I think this would look better if we change `aplugin` to `plugin_name` and then remove the name from the warning string. Instead it could be something like `This plugin is using a deprecated RPC protocol.`. I'm not convinced if we should have some sort of message pointing them somewhere (a url?) where they can find out more. What do you think?